src/Symfony/Component/Webhook/Controller/WebhookController.php

```diff
@@ -40,12 +40,17 @@ public function handle(string $type, Request $request): Response
         }
         /** @var RequestParserInterface $parser */
         $parser = $this->parsers[$type]['parser'];
+        $events = $parser->parse($request, $this->parsers[$type]['secret']);
 
-        if (!$event = $parser->parse($request, $this->parsers[$type]['secret'])) {
+        if (null === $events) {
```

Comment:
I think an empty array (i.e. ignoring all events) should be treated as a rejected response too